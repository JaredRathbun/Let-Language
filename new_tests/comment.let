# Calculates the head of two lists added #

let lst1, lst2 := list(1, 2, 3), list(4, 5, 6) in hd lst1 + hd lst2