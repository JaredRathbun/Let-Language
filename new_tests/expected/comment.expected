Parse successful!
Result: 5